knox/settings.py

```diff
@@ -40,8 +43,9 @@ class CONSTANTS:
     '''
     Constants cannot be changed at runtime
     '''
-    TOKEN_KEY_LENGTH = 8
+    TOKEN_KEY_LENGTH = 15
```

Comment:
I’ve extended the length here a bit to consider a potential token
WDYT security wise?